CHANGES.md

```diff
@@ -1,10 +1,12 @@
 ## Version 11.7.0
 
 Grammars:
-- Added 3rd party LookML grammar to SUPPORTED_LANGUAGES [Josh Temple][]
+- enh(js) added `import` operator 
```

Comment:
Please add credit here and a link below (line ~7)...

Reply:
Like this?